packages/ember-engines/addon/components/link-to-external.js

```diff
@@ -13,6 +13,10 @@ if (gte('ember-source', '3.24.0-alpha.1')) {
 
       return externalRoute;
     }
+
+    // override LinkTo's assertLinkToOrigin method to noop. In LinkTo, this assertion
+    // checks to make sure LinkTo is not being used inside a routeless engine
+    assertLinkToOrigin() {}
```

Comment:
Can you update this comment to link to the PR over in Ember?

Reply:
done